test_scripts/API/ServiceStatusUpdateToHMI/002_3_RPC_service_unprotected.lua

```diff
@@ -0,0 +1,71 @@
+-----------------------------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0211-ServiceStatusUpdateToHMI.md
+-----------------------------------------------------------------------------------------------------------------------
+-- Description: Opening of the non-protected Audio/Video/RPC service with successful OnServiceUpdate notification
+--
+-- Preconditions:
+-- 1) SDL certificate is missing/expired
+-- 2) App is registered with NAVIGATION appHMIType and activated
+-- Steps:
+-- 1) App sends StartService request (<service_type>, encryption = false)
+-- SDL does:
+--   - send OnServiceUpdate (<service_type>, REQUEST_RECEIVED) to HMI
+--   - not send GetSystemTime() request to HMI
+--   - not start PTU sequence
+--   - send OnServiceUpdate (<service_type>, REQUEST_ACCEPTED) to HMI
+--   - send StartServiceACK(<service_type>, encryption = false) to App
+-----------------------------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ServiceStatusUpdateToHMI/common')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+function common.startServiceFunc(pServiceId)
+  local msg = {
+    frameType = common.const.FRAME_TYPE.CONTROL_FRAME,
+    serviceType = pServiceId,
+    frameInfo = common.const.FRAME_INFO.START_SERVICE,
+    encryption = false
+  }
+  common.getMobileSession():Send(msg)
+end
+
+function common.serviceResponseFunc(pServiceId)
+  common.getMobileSession():ExpectControlMessage(pServiceId, {
+    frameInfo = common.const.FRAME_INFO.START_SERVICE_ACK,
+    encryption = false
+  })
+end
+
+function common.policyTableUpdateFunc()
+  -- common.getHMIConnection():ExpectNotification("SDL.OnStatusUpdate")
+  -- :Times(0) -- There is an issue: SDL sends 'UPDATE_NEEDED' if it was build in EXTERNAL_PROPRIETARY mode
```

Comment:
Link to issue?

Reply:
@jacobkeeler Please find link to issue in e752836